packages/browser/src/transports/fetch.ts

```diff
@@ -25,6 +25,12 @@ export class FetchTransport extends BaseTransport {
         : '') as ReferrerPolicy,
     };
 
-    return (global as Window).fetch(this.url, defaultOptions);
+    const response = await (global as Window).fetch(this.url, defaultOptions);
```

Comment:
It'll break if `fetch` throws. We should put it in `try/catch` clause I guess

Reply:
Or will `async` wrap it in `reject` call? 🤔

Reply:
`async` handles everything 💪